tests/test_models.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import absolute_import, unicode_literals
+
+from django.test import TestCase
+from wagtailcaptcha.forms import WagtailCaptchaFormBuilder
+
+from home.models import TestCaptchaEmailFormPage, TestCaptchaFormPage
+
+
+class TestCaptchaEmailFormPageTestCase(TestCase):
```

Comment:
I purposely left out the test for `process_form_submission` as it is to be tackled in #11.